spec/features/idv/doc_auth/address_step_spec.rb

```diff
@@ -42,22 +63,40 @@
     end
   end
 
-  context 'with Puerto Rico address' do
+  context 'with a Puerto Rico address' do
     before do
       sign_in_and_2fa_user
       complete_doc_auth_steps_before_document_capture_step
       complete_document_capture_step_with_yml('spec/fixtures/puerto_rico_resident.yml')
       complete_ssn_step
     end
 
-    it 'shows address guidance and hint text' do
+    it 'renders the Puerto Rico address guidance as visible text' do
+      guidance = page.find_by_id('puerto-rico-extra-text')
+
+      expect(guidance.text).to include(puerto_rico_guidance_text)
+      expect(guidance).to be_visible
+    end
+
+    it 'accepts a Puerto Rico style address' do
       expect(page).to have_current_path(idv_address_url)
-      expect(page.body).to include(t('doc_auth.info.address_guidance_puerto_rico_html'))
       expect(page).to have_content(t('forms.example'))
       fill_in 'idv_form_address1', with: '123 Calle Carlos'
       fill_in 'idv_form_address2', with: 'URB Las Gladiolas'
       click_button t('forms.buttons.submit.update')
       expect(page).to have_current_path(idv_verify_info_path)
     end
+
+    context 'that gets changed to a mainland address' do
+      before do
+        select 'Wyoming', from: 'idv_form_state'
+      end
+
+      it 'hides the Puerto Rico address guidance' do
+        guidance = page.find_by_id('puerto-rico-extra-text', visible: false)
```

Comment:
This also needs to check that the hint text is hidden.

Reply:
It did (the `visible: false` implies that, but is not the clearest thing in the world). No longer matters, as I re-factored this spec per your and Andrew's suggestions.